src/main/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/postsubmit/AppealPaymentConfirmation.java

```diff
@@ -43,6 +43,8 @@ public PostSubmitCallbackResponse handle(
             new PostSubmitCallbackResponse();
 
         final AsylumCase asylumCase = callback.getCaseDetails().getCaseData();
+        final String feeString = "\n\n#### Fee\n";
+        final String paymentReferenceNumber = "\n\n#### Payment reference number\n";
```

Comment:
I think using this variable name could be misleading. Possibly look to use the same naming convention as you used above for `feeString` (i.e. paymentReferenceNumberString or paymentReferenceNumberTitle). Alternatively define a CONST field for these repeated String usages.

Reply:
Renamed to feeLabel and paymentReferenceNumberLabel